package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "mdn-translated-content",
+  "version": "1.0.0",
+  "description": "Translated content of https://developer.mozilla.org",
+  "private": true,
+  "scripts": {
+    "lint:md": "markdownlint \"**/*.md\" -i node_modules"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mdn/translated-content.git"
+  },
+  "keywords": [],
+  "author": "",
```

```suggestion
  "author": "MDN Web Docs",
```
if we are providing this field then it could be like what yari has